sdcm/sct_config.py

```diff
@@ -1222,6 +1222,10 @@ class SCTConfiguration(dict):
         dict(name="availability_zone", env="SCT_AVAILABILITY_ZONE",
              type=str,
              help="Availability zone to use. Same for multi-region scenario."),
+        dict(name="auto_availability_zone", env="SCT_AWS_AUTO_AVAILABILITY_ZONE",
```

Comment:
Having env var be called `SCT_AWS_AUTO_AVAILABILITY_ZONE` the python var must be named as `aws_auto_availability_zone`.

Reply:
fixed